cpp/src/parquet/metadata.h

```diff
@@ -282,11 +282,20 @@ class PARQUET_EXPORT FileMetaData {
 
   bool Equals(const FileMetaData& other) const;
 
-  /// \brief The number of top-level columns in the schema.
+  /// \brief The number of parquet "leaf" fields.
```

Comment:
In the parquet specs, the official wording is `column`. 

I'd suggest follow the same syntax.

https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L680

Reply:
Yes, it can't hurt to be coherent. I made the change.